org.eclipse.dawnsci.hdf5.test/src/org/eclipse/dawnsci/hdf5/TestPlugins.java

```diff
@@ -62,7 +64,14 @@ public void testBSFilter() throws NullPointerException, HDF5Exception {
 		testFilter("test-scratch/testFilterBS.h5", 32008, 0, 2);
 	}
 
+	private static void assertFilterAvailable(int filterNumber) throws HDF5LibraryException, NullPointerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.NullPointerException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)